README.md

```diff
@@ -20,7 +20,7 @@ How To Use
   <dependency>
     <groupId>com.simple</groupId>
     <artifactId>jdub_${scala.major.version}</artifactId>
-    <version>0.9.0</version>
+    <version>${jdub.version}</version>
```

Comment:
I'm not sure most projects will have this defined (whereas we know the Scala version will be).